src/main/java/me/partlysanestudios/partlysaneskies/config/oneconfig/OneConfigScreen.java

```diff
@@ -1265,6 +1265,72 @@ public void resetBrokenStringsTick() {
     )
     public boolean visitorLogbookStats = false;
 
+//    Wrong Tool for Crop
+    @Switch(
+            subcategory = "Wrong Tool for Crop",
+            name = "Wrong Tool for Crop Enabled",
+            description = "When enabled, a warning will appear, notifying you that you are using the wrong tool for the current crop",
+            category = "Farming"
+    )
+    public boolean wrongToolForCropEnabled = false;
+
+    @Switch(
+            subcategory = "Wrong Tool for Crop",
+            name = "Allow Mathematical Hoes",
+            description = "When enabled, mathematical hoes will be considered a valid tool",
```

Comment:
Why is this even an option

Reply:
Why not

Reply:
This should be the default behaviour, i dont see anyone disabling that

Reply:
It's more like, that way every use can see exactly what will trigger, because descriptions are hidden until you hover